sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -18,14 +18,15 @@
 import io.opentelemetry.sdk.trace.SpanProcessor;
 import io.opentelemetry.sdk.trace.data.SpanData;
 import java.util.ArrayList;
-import java.util.Collections;
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.BlockingQueue;
+import java.util.Collection;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.ReentrantLock;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import org.jctools.queues.MpscArrayQueue;
```

Comment:
Is this better than `disruptor`? Can we run the same benchmark for https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/async-processor ?

Reply:
@bogdandrutu Is there an existing jmh benchmark with DisruptorAsyncSpanProcessor?

Reply:
Unfortunately not, but I think you can copy-paste the same one that you have and only initialize that SpanProcessor.